au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TrackItem.qml

```diff
@@ -174,9 +174,11 @@ ListItemBlank {
         SeparatorLine {}
 
         Row {
-            Layout.fillHeight: true
+            Layout.alignment: Qt.AlignTop
             Layout.preferredWidth: childrenRect.width
-            Layout.margins: 8
+            Layout.margins: 5
+            Layout.bottomMargin: 20
+            height: root.height
```

Comment:
it is the same as `Layout.fillHeight: true`, isn't it?

Reply:
Okay, I hope this will be fixed within #6768 and we won't need the clip at all
By the way, please replace `height` with `Layout.preferedHeight` - if the object is inside the layout, then the anchors and sizes should be set via the Layout

Reply:
Done